src/touringcar/models.py

```diff
@@ -37,7 +37,7 @@ class Meta:
     enddate = models.DateField(help_text="Publicatie einddatum")
     category = models.CharField(max_length=55, blank=True, null=True)
     link = models.URLField(max_length = 200, blank=True, null=True)
-    image_url = models.ImageField(upload_to='images', blank=True, null=True)
+    image_url = models.ImageField(upload_to='touringcar-images', blank=True, null=True)
```

Comment:
Is this right? Do we not need another change somewhere else that reads from this directory?

Reply:
this is right, the front-end reads